lib/rewards/server/message-actions.js

```diff
@@ -11,46 +11,77 @@ class RewardMessageActions
             return false;
         }
         if (ObjectsConst.OBJECT_INTERACTION !== message.act ||
-            RewardsConst.REWARDS_PICK_UP_ACT !== message.type) {
+            RewardsConst.REWARDS_PICK_UP_ACT !== message.type ||
+            !message['id']) {
```

Comment:
message.id, we don't need the braces for a simple property

Reply:
also: 
- do an early return for the message.type
- then assign rewardId using: `let rewardId = sc.get(message, 'id', false)`
- add a second early return if `false === rewardId`

Reply:
done!